### DISCUSSION TITLE
issue with running jamulus headless on a VCS

### DISCUSSION THREAD

**Discussion by WernerBleisteiner** [jamulussoftware/jamulus#1200] [Documentation]
Hi!
I'm seem to miss something when setting up Jamulus headless server on a VCS (Ubuntu 20.4)
Compilation and installation according to the guide[](https://jamulus.io/wiki/Server-Linux) was easy. Also includes the systemd start script.
After launch status tells me Jamulus is running - but registration to the Central server fails. Apparently as there's no free slot.

Alternatively I tried to make my server non public and then connect to it directly via IPv4, either w/o port numbers. But the clients GUI only gets as far as "connecting" (Verbindungsaufbau) .Tried also with both methods:  CLI start or adapted systemd script.
Any hints, please?

**Comment by dcorson-ticino-com** [comment]
Hi Werner,
Start out by registering to a different central server that is less full.
The possibilities are to be seen in the command line doc:
https://jamulus.io/wiki/Command-Line-Options

You probably just have the wrong IP for connecting to our private server.
Once you are up public you can use the Jamulus Explorer to find your server and the IP address.
https://explorer.jamulus.io/

**Reply by WernerBleisteiner** [reply]
Thanks. Now I succeeded. Omitted the systemd and launched via CLI, after looking for an available slot.
Problem was obviously that the systemd unit file instruction refers to just one, the 'old' server.
Look forward to jam with some friends...

**Comment by ann0see** [comment]
Sorry if I'm off topic here but you might soon be able to install the headless server via a .deb file. If you do this once, I'd be glad about some feedback.

**Reply by WernerBleisteiner** [reply]
Thanks. I succeeded now. (see above)
However, a .deb file would certainly ease the set-up.
